web/models/release.ex

```diff
@@ -10,7 +10,7 @@ defmodule HexWeb.Release do
     timestamps
 
     belongs_to :package, Package
-    has_many :requirements, Requirement
+    has_many :requirements, Requirement, on_replace: :delete
```

Comment:
Would it be better to explicitly delete requirements instead? When is this needed?